applications/ParticleMechanicsApplication/custom_constitutive/flow_rules/mc_plastic_flow_rule.cpp

```diff
@@ -77,7 +77,7 @@ MCPlasticFlowRule::~MCPlasticFlowRule()
 {
 }
 
-void MCPlasticFlowRule::InitializeMaterial(YieldCriterionPointer pYieldCriterionPointer, HardeningLawPointer pHardeningPointer, const Properties& rProp)
+void MCPlasticFlowRule::InitializeMaterial(YieldCriterionPointer& pYieldCriterionPointer, HardeningLawPointer& pHardeningPointer, const Properties& rProp)
```

Comment:
This FlowRule class disappears and other classes are templates in the new ConstitutiveModelsApplication